docs/features/feature-app-size-monitoring.md

```diff
@@ -27,4 +27,5 @@ Read more about AAB format [here](https://developer.android.com/guide/app-bundle
 
 ## iOS
 
-Coming soon...
+Measure supports tracking the size of the IPA file. This represents the size of the generated .ipa, not the actual App Store download size.
+App size tracking is handled via the [upload_dsyms.sh](../../ios/Scripts/upload_dsyms.sh) script.
```

Comment:
Measure automatically tracks size of the IPA file for each version when you upload dSYMs via the [upload_dsyms.sh](../../ios/Scripts/upload_dsyms.sh) script.

> [!NOTE]
> This represents the size of the generated .ipa, not the actual App Store download size.